### ISSUE TITLE
Add Image

### ISSUE THREAD

**Issue by athirasanthosh** [JeasonWong/Particle#9]
Is it possible to change particle text as image

**Comment by itfdw** [comment]
yes like an logo , there is a way?

**Comment by athirasanthosh** [comment]
yes. when i open an application, i need to give this effect to logo(Image). Is it possible?
